common/changes/@rushstack/heft-jest-plugin/add-node-env-config-to-heft-jest-plugin_2023-09-26-12-02.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-jest-plugin",
+      "comment": "add new test that covers setting NODE_ENV var if not set",
```

```suggestion
      "comment": "Ensure that the NODE_ENV environment variable is set correctly when running Jest tests",
```

The unit test is not interesting to package consumers, but the environment variable behavior is.